lib/src/components/savings_account_sum_component.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:flame/components.dart';
+import 'package:flame_riverpod/flame_riverpod.dart';
+import 'package:investania/src/providers/savings_account_provider.dart';
+
+class SavingsAccountSumComponent extends Component with HasComponentRef {
```

Comment:
Since Teddy is using `HasComponentReference` we can use that here too, instead of `gameRef` it is then called `game`. It's the new mixin to use.

Reply:
Maybe call it just `SavingsAccountSum` since the other components aren't suffixed with `Component`.